include/envoy/stream_info/stream_info.h

```diff
@@ -201,6 +201,15 @@ class StreamInfo {
    */
   virtual void onUpstreamHostSelected(Upstream::HostDescriptionConstSharedPtr host) PURE;
 
+  /**
+   * @param name denotes the name of the route.
+   */
+  virtual void setRouteName(std::string name) PURE;
+
+  /**
+   * @return the name of the route.
```

Comment:
Please add the type to the @return comment (`@return std::string& the name of the route`)